pkgs/development/libraries/liquid-dsp/default.nix

```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "liquid-dsp-${version}";
+  version = "20170307";
+
+  src = fetchFromGitHub {
+    owner = "jgaeddert";
+    repo = "liquid-dsp";
+    rev = "8c1978fa4f5662b8849fe712be716958f29cec0e";
```

Comment:
v1.3.0 fails to compile:

```
/nix/store/ndqahbwrnabiaszk0s8cblhgwhid5kn0-gcc-5.4.0/lib/gcc/x86_64-unknown-linux-gnu/5.4.0/include/pmmintrin.h:50:1: error: inlining failed in call to always_inline '_mm_addsub_ps': target specific option mismatch
 _mm_addsub_ps (__m128 __X, __m128 __Y)
 ^
src/dotprod/src/dotprod_cccf.mmx.c:255:11: error: called from here
         s = _mm_addsub_ps( ci, cq );
           ^
In file included from src/dotprod/src/dotprod_cccf.mmx.c:49:0:
```

This seems to be fixed in the version I referenced.